qt/python/.gitignore

```diff
@@ -0,0 +1,107 @@
+# File generated by the build
+mantidqt/utils/qt/plugins.py
```

Comment:
Why not just add these to the top level `.gitignore`?

Reply:
Good point. It saves confusion over where these things are.